### PR TITLE
Appsec/anna/v0 integrationtests iisfixture sharing

### PR DESCRIPTION
**Pull request by anna-git**
Fixes #

Changes proposed in this pull request:
Sharing IisFixture and GacFixture moved toTestHelpers to be used by Security integration tests too 

@DataDog/apm-dotnet

### CODE REVIEW

File: devenv.bat [hunk 1]
```diff
@@ -54,6 +54,7 @@ SET CORECLR_ENABLE_PROFILING=1
 SET CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
 SET CORECLR_PROFILER_PATH=%~dp0\src\Datadog.Trace.ClrProfiler.Native\bin\%profiler_configuration%\%profiler_platform%\Datadog.Trace.ClrProfiler.Native.dll
 set DD_TRACE_CALLTARGET_ENABLED=0
+set DD_APPSEC_ENABLED=0
```
Comment (robertpi) [inline devenv.bat:57]: Why add this here when it's off by default?

File: test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs [hunk 2]
```diff
@@ -3,188 +3,20 @@
...
-using Datadog.Trace.TestHelpers;
+using Datadog.Trace.ClrProfiler;
 using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
 [assembly: TestFramework("Datadog.Trace.ClrProfiler.IntegrationTests.CustomTestFramework", "Datadog.Trace.ClrProfiler.IntegrationTests")]
 
-namespace Datadog.Trace.ClrProfiler.IntegrationTests
+namespace Datadog.Trace.Security.IntegrationTests
```
Comment (robertpi) [inline test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs:13]: Are you sure the namespace should change?

File: test/test-applications/throughput/Samples.AspNetCoreSimpleController/Samples.AspNetCoreSimpleController.csproj [hunk 3]
```diff
@@ -5,4 +5,8 @@
     <Platforms>x64;x86;AnyCPU</Platforms>
   </PropertyGroup>
 
+  <ItemGroup>
```
Comment (robertpi) [inline test/test-applications/throughput/Samples.AspNetCoreSimpleController/Samples.AspNetCoreSimpleController.csproj:8]: Why is this needed to be referenced by the samples now?

**Review by robertpi — COMMENTED** [review]
Couple of minor questions / improvements but otherwise LGTM.

File: devenv.bat (hunk 1 above)
Comment (anna-git) [reply devenv.bat:57]: just that my memory is bad, and I always have to look it up in our project to activate it here, so here just have to turn to 1^^  but can remove!

File: test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs (hunk 2 above)
Comment (anna-git) [reply test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs:13]: good point, I mixed them up!

File: test/test-applications/throughput/Samples.AspNetCoreSimpleController/Samples.AspNetCoreSimpleController.csproj (hunk 3 above)
Comment (anna-git) [reply test/test-applications/throughput/Samples.AspNetCoreSimpleController/Samples.AspNetCoreSimpleController.csproj:8]: I guess there was a rebase error at some point but it couldnt compile because of Program.cs line 12, using the Instrumentation type, this reference is on master..

**Comment by anna-git** [comment]
Was merged directly to main branch
